test/parallel/test-repl-import-referrer.js

```diff
@@ -27,5 +27,9 @@ child.on('exit', common.mustCall(() => {
   );
 }));
 
-child.stdin.write('await import(\'./message.mjs\');\n');
-child.stdin.write('.exit');
+// Note: write the commands on stdin with a slight delay to make sure
+//       that the child process is ready to receive and process them
+setTimeout(() => {
+  child.stdin.write('await import(\'./message.mjs\');\n');
+  child.stdin.write('.exit');
+}, 250);
```

Comment:
Why 250? Did you try other values? Could we use a lower one? We should probably be using `common.platformTimeout()` btw

Reply:
> Why 250? Did you try other values? Could we use a lower one? 

250 is totally arbitrary 😓 , I did try different values, `100` seems to be working fine on my
machine but I increased the value as I think a bigger delay would probably be needed in CI

would you prefer me to put `100` instead and see how this performs in CI?

> We should probably be using common.platformTimeout() btw

ah yeah for sure, sorry I forgot about this 😓